cvat-ui/src/components/annotation-page/appearance-block.tsx

```diff
@@ -168,6 +180,29 @@ function AppearanceBlock(props: Props): JSX.Element {
                             min={0}
                             max={100}
                         />
+                        {is3D && (
+                            <div className='cvat-appearance-arrows-checkboxes'>
+                                <div>
+                                    <Text type='secondary'>Orientation arrows</Text>
+                                </div>
+                                <div>
+                                    {
+                                        Object.keys(showAxisArrows).map((axis) => (
```

Comment:
IMO we do not need to control each axis visibility
It makes things more difficult and I do not know who needs that. 

Just one checkbox to decide all visible or all invisible. 
I believe for the most of users it will be more convenient to click just one checkbox also.